django_project/lesson/tests/test_section_view.py

```diff
@@ -214,10 +222,12 @@ def test_SectionDelete_with_login(self):
             response,
             reverse(
                 'section-list',
-                kwargs={'project_slug': section_to_delete.project.slug}
+                kwargs={'project_slug':
+                            section_to_delete.project.slug}
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->